tests/test_rawpy.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for rawpy plugin
+"""
+
+import rawpy
+import imageio.v3 as iio
+import numpy as np
+
+
+def test_nef_local(test_images):
+    """Test for reading .nef file from .test_images dir.
+    """
+    # Construct image path
+    im_path = "e:/Explorations/imageio-tahseen-new/.test_images/infrared.nef" #test_images / "infrared.nef"
```

Comment:
How come you've replaced this with a path local to your filesystem?

Reply:
I forgot to revert the change before the commit. I replaced this with a path local to my filesystem because when I use the `test_images` parameter, I get the following error:

`e:\Explorations\imageio-tahseen-new\tests\test_rawpy.py::test_nef_local failed: TypeError: bad argument type for built-in operation

The above exception was the direct cause of the following exception:

test_images = WindowsPath('e:/Explorations/imageio-tahseen-new/.test_images')

    def test_nef_local(test_images):
        """Test for reading .nef file from .test_images dir.
        """
        # Construct image path
        im_path = test_images / "infrared.nef"
    
        # Test if plugin's content mathces rawpy content
        actual = iio.imread(im_path, plugin="rawpy")
>       expected = rawpy.imread(im_path).postprocess()

tests\test_rawpy.py:17: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.venv\Lib\site-packages\rawpy\__init__.py:20: in imread
    d.open_file(pathOrFile)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   SystemError: <class 'RuntimeError'> returned a result with an exception set

rawpy\\_rawpy.pyx:404: SystemError`

So, I replaced the value with a local path for quick test. Do you know why I am seeing this error ? Is it because of my OS ?

Reply:
Oh, that looks like a `String` vs `pathlib.Path` error. can you try again with

```python
im_path = test_images / "infrared.nef"
actual = iio.imread(str(im_path), plugin="rawpy")
```

If this works, then we have to update the plugin, because ImageIO also allows `pathlib.Path` inputs.

Reply:
Hi, the issue is not happening with imageio plugin. It happens when I try to directly read the image using rawpy in my test. The error is in the line `expected = rawpy.imread(im_path)` and not `iio.imread(im_path, plugin="rawpy")` . I have updated the code to `expected = rawpy.imread(str(im_path))` and now it is working.